cmake/zbarConfig.cmake

```diff
@@ -0,0 +1,5 @@
+include(CMakeFindDependencyMacro)
+
+find_dependency(Threads)
```

Comment:
no settings for pthreads here? What if the library was built without threads? As it is the consumer would need to find it.

Reply:
How can I save if the package was built with or without threads?

Reply:
Changed to only find it if built with Threads